modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -94,6 +95,9 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::validParams()
       "old_creep_strain_forcing_function",
       "Advanced");
 
+  params.addParam<std::string>("stress_unit", "Pa", "unit of stress");
```

Comment:
I think this is a great improvement. These comments are just questions and not a request for any code changes.
This unit is for the units used in the elastic material block.  Is there a way to make sure the elastic material and this material use the same units or to get the units from the elastic materials block.  Maybe the parameter could be called stressUnitsFromYourElasticMaterialsBlock but that is too long.

Reply:
> This unit is for the units used in the elastic material block. Is there a way to make sure the elastic material and this material use the same units or to get the units from the elastic materials block.

I think other tensor mechanics objects don't store any information about unit. We just rely on users to make units consistent. So there isn't really a way to infer units from other objects.

> Maybe the parameter could be called stressUnitsFromYourElasticMaterialsBlock but that is too long.

I think the parameter names aren't supposed to be self-explanatory, are they? We use the `doc_string` to add descriptions -- I'll elaborate the `doc_string` for this parameter.